<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QDialog" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#admin
{
background:rgb(231, 227, 247);
}
#back
{
background:url(:/first/user_m.jpg);
}
QFrame
{
background:rgb(48, 64, 66);
border-color:rgb(48, 64, 66);
}
#welcom
{
background : transparent;
border: none;
font:12pt &quot;Papyrus&quot;;
color:rgb(203, 213, 255);
}
#search
{
background:transparent;
border: 1px solid #dddbff;
color:#dddbff;
	font: 14pt &quot;Papyrus&quot;;
}
QRadioButton
{
color:#dddbff;
	font: 10pt &quot;Papyrus&quot;;

}
#show
{
font:10pt &quot;Papyrus&quot;;
background:rgba(255, 255, 0, 0.5);
}

#profile
{
background:rgb(235, 22, 22);
border-radius:20px;
}
#profile:hover
{
background:rgb(0, 255, 0);
}
QPushButton
{
font:12pt &quot;Papyrus&quot;;
background:#c64545;
border : 1px solid #9c1c1c;

}
QPushButton:hover
{
background:#ffff00;
}
QLineEdit
{
background:transparent;
border:none;
color:#c64545;
border-bottom:1px solid #c64545;
font:15pt &quot;Papyrus&quot;;
}
#show_2
{
font:10pt &quot;Papyrus&quot;;
background:rgba(255, 255, 0, 0.5);
font-size:15px;
}
#exit
{
background : transparent;
	font: 17pt &quot;MV Boli&quot;;
color:red;
border:none;
}
#white
{
background:rgb(255, 255, 255);
}
#white_2
{
background:rgb(255, 255, 255);
}
#exit:hover
{
color:#379eff;
}
#amanat
{
border:none;
background:#c64545;
border : 1px solid #9c1c1c;
}
#all
{
border:none;
background:#c64545;
border : 1px solid #9c1c1c;
}
#show_3
{
font:10pt &quot;Papyrus&quot;;
background:rgba(255, 255, 0, 0.5);
font-size:15px;
}
#white_3
{
background:rgb(255, 255, 255);
}
#guid
{
font: 9pt &quot;MS Serif&quot;;
color:rgb(255, 255, 255);
background:#c64545;
border : 1px solid #9c1c1c;
}
#guid_2
{
font: 9pt &quot;MS Serif&quot;;
color:rgb(255, 255, 255);
background:#c64545;
border : 1px solid #9c1c1c;
}
#guid_3
{
font: 9pt &quot;MS Serif&quot;;
color:rgb(255, 255, 255);
background:#c64545;
border : 1px solid #9c1c1c;
}
#amanat
{
color:white;
	font: 11pt &quot;MV Boli&quot;;
setAlignment:center;
}</string>
  </property>
  <widget class="QLineEdit" name="publisher">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit
{
color:#c3b9bf;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Publisher...</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="show_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Papyrus'; font-size:15px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="profile">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/first/login.png</normaloff>:/first/login.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="group">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>group</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="name_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcomlable">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#welcomlable
{
background:transparent;
color:rgb(202, 189, 174);
font: 15pt &quot;Papyrus&quot;;
}</string>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit
{
color:#c3b9bf;
}</string>
    </property>
    <property name="placeholderText">
     <string> search...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="author_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>author</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="publisher_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>publisher</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="num">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Number...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="author">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Author...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Name...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="group3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit
{
color:#c3b9bf;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Group...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="group2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit
{
color:#c3b9bf;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Group...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="group1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit
{
color:#c3b9bf;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Group...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>add book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>edit book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>remove book</string>
   </property>
  </widget>
  <widget class="QFrame" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>601</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="view">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>view book list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_g">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>view group list</string>
    </property>
   </widget>
   <widget class="QFrame" name="white_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>-10</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextBrowser" name="show">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>401</width>
       <height>171</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Papyrus'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="amanat">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MV Boli'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string extracomment="borrowed books"/>
    </property>
   </widget>
   <widget class="QFrame" name="white_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextBrowser" name="show_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Papyrus'; font-size:15px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="guid">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#welcomlable
{
background:transparent;
color:rgb(202, 189, 174);
font: 15pt &quot;Papyrus&quot;;
}</string>
    </property>
    <property name="text">
     <string>for add you must enter name,author,publisher</string>
    </property>
   </widget>
   <widget class="QLabel" name="guid_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#welcomlable
{
background:transparent;
color:rgb(202, 189, 174);
font: 15pt &quot;Papyrus&quot;;
}</string>
    </property>
    <property name="text">
     <string>for remove you must enter the number in list</string>
    </property>
   </widget>
   <widget class="QLabel" name="guid_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#welcomlable
{
background:transparent;
color:rgb(202, 189, 174);
font: 15pt &quot;Papyrus&quot;;
}</string>
    </property>
    <property name="text">
     <string>for edit you must enter the number in list</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="amanat_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MV Boli'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string extracomment="borrowed books"/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="all">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MV Boli'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string extracomment="borrowed books"/>
    </property>
   </widget>
   <zorder>amanat</zorder>
   <zorder>white_2</zorder>
   <zorder>view</zorder>
   <zorder>view_g</zorder>
   <zorder>white_3</zorder>
   <zorder>guid</zorder>
   <zorder>guid_2</zorder>
   <zorder>guid_3</zorder>
   <zorder>amanat_2</zorder>
   <zorder>all</zorder>
  </widget>
  <widget class="QFrame" name="white">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>back</zorder>
  <zorder>white</zorder>
  <zorder>show_2</zorder>
  <zorder>publisher</zorder>
  <zorder>frame</zorder>
  <zorder>num</zorder>
  <zorder>author</zorder>
  <zorder>name</zorder>
  <zorder>group3</zorder>
  <zorder>group2</zorder>
  <zorder>group1</zorder>
  <zorder>add</zorder>
  <zorder>edit</zorder>
  <zorder>remove</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
